routes/api.php

```diff
@@ -35,3 +41,13 @@
         Route::post('/', StoreUserController::class);
         Route::delete('/{user}', DeleteUserController::class);
     });
+
+Route::prefix('cities')
+    ->middleware([])
+    ->group(static function () {
+        Route::get('/', ListCityController::class);
+        Route::get('/{city}', GetCityController::class)->withTrashed();
```

Comment:
I don't like the withTrashed here in the API, I feel you could do it in other place